File: qlty-cli/tests/lang/typescript/basic.in/lcom/lcom_2.ts
// lcom=2

class KlassA {
  constructor() {}

  foo(): any  {
    return this.aaa
  }

  bar(): any  {
    return this.bbb
  }
}

class KlassB {
  constructor() {}

  foo(): any  {
    return this.baz
  }

  bar(): any  {
    return this.baz
  }
}

File: qlty-cli/tests/lang/swift/basic.in/lcom/lcom_2.swift
// lcom=2

class KlassA {
    private var aaa: String = "aaa"
    private var bbb: String = "bbb"
    
    func getBbb() -> String {
        return bbb
    }
    
    func getAaa() -> String {
        return aaa
    }
    
    func foo() -> String {
        return getAaa()
    }
    
    func bar() -> String {
        return getBbb()
    }
}

class KlassB {
    private var baz: String = "baz"
    
    func getBar() -> String {
        return baz
    }
    
    func foo() -> String {
        return getBar()
    }
    
    func bar() -> String {
        return getBar()
    }
}